proto/v1/event.proto

```diff
@@ -67,9 +67,10 @@ message EventCollectionRequest {
   // - discoveryDate
   // - occurrenceTime
   string date = 3;
+  double fire_event_area_limit = 4;
```

Comment:
@chejennifer Please set this value in frontend request. It filters out events with smaller area.